docs/source/installation.rst

```diff
@@ -6,6 +6,9 @@ following pages have instructions that are specific to your system.  Once you
 are done installing the dependencies, come back to this page to install manim
 itself.
 
+.. tip::
+    In case that you want to try manim online without installation, open it in `Binder <https://mybinder.org/v2/gist/behackl/725d956ec80969226b7bf9b4aef40b78/HEAD?filepath=basic%20example%20scenes.ipynb>`_
```

Comment:
@behackl : currently the binder is in your public gist.
I think that is totally fine, however, it is also possible to launch notebooks from github repos, so why not do a new repo in the organization for that?
Also, I already created a link for a notebook to binder in the twitter repo: 
https://github.com/ManimCommunity/manim-tweets
So maybe we can use this repo for example notebooks.

Reply:
I think preparing a repository such that binder links can be used from there is a good idea.

Reply:
new place is now here:
https://mybinder.org/v2/gh/ManimCommunity/jupyter_examples/HEAD?filepath=basic_example_scenes.ipynb